dags/fivetran_acoustic.py

```diff
@@ -10,6 +10,9 @@
 
 
 def _generate_acoustic_report(conn_id, report_type, config, *args, **kwargs):
```

Comment:
Docstrings? Or what is our current practice for code documentation: functions & arguments data types and descriptions?